github/ci/prow-deploy/files/jobs/kubevirt/containerdisks/containerdisks-presubmits.yaml

```
@@ -87,6 +87,42 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    optional: false
+    annotations:
+      testgrid-create-test-group: "false"
+    cluster: prow-s390x-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-podman-in-container-enabled: "true"
+    max_concurrency: 11
+    name: pull-containerdisks-pipeline-centos-stream-9-s390x
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - ./pipeline.sh
+        env:
+        - name: GIMME_GO_VERSION
+          value: "1.22.9"
+        - name: FOCUS
+          value: "centos-stream:9"
+        - name: KUBEVIRTCI_TAG
+          value: "2501140834-56eb34e6"
+        - name: SLIM
```

I think it is : 
```suggestion
        - name: KUBEVIRT_SLIM
```

But I might be wrong.

---

I commited this change, let's see if it works now. Thanks for your support here!